ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```
@@ -1668,6 +1668,30 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-upi-private-xpn-sdn
+- as: ocp-e2e-ibmcloud-ovn-multi-ppc64le-cucushift
+  cron: 20 17 * * *
+  steps:
+    cluster_profile: ibmcloud-multi-ppc64le
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_ARCHITECTURE: ppc64le
+      BASE_DOMAIN: ocp-dev-ppc64le.com
+      E2E_RUN_TAGS: '@ppc64le and @ibmcloud-ipi and @heterogeneous and @network-ovnkubernetes
+        and not @fips'
+      E2E_SKIP_TAGS: not @upgrade and not @customer and not @destructive and not @flaky
+        and not @inactive and not @long-duration and not @prod-only and not @qeci
+        and not @security and not @stage-only and not @upgrade-check and not @upgrade-prepare
```

```suggestion
      E2E_RUN_TAGS: '@ppc64le and @ibmcloud-ipi and @heterogeneous and @network-ovnkubernetes
        and not @fips'
      E2E_SKIP_TAGS: not @upgrade and not @customer and not @destructive and not @flaky
        and not @inactive and not @long-duration and not @prod-only and not @qeci
        and not @security and not @stage-only and not @upgrade-check and not @upgrade-prepare
```

Please go ahead and remove this per guidance

---

I had made the changes . please review